internals/ci/src/cmds/version/index.ts

```diff
@@ -8,41 +9,93 @@ import { expectSuccess } from "@oko-wallet-ci/expect";
 import { sleep } from "@oko-wallet-ci/time";
 import { doBuildSDK } from "../build_sdk";
 
+function findPackageJsonFiles(dir: string): string[] {
+  const results: string[] = [];
+  const entries = fs.readdirSync(dir, { withFileTypes: true });
```

Comment:
For entries, if you think about it, we use lerna.json, and it has all the packages we'll publish already. No need to search the entire repository I guess?